content/spin/v2/serverless-ai-hello-world.md

```diff
@@ -188,22 +187,22 @@ watch = ["src/index.ts"]
 <!-- @selectiveCpy -->
  
 ```toml
-spin_manifest_version = "1"
-authors = ["Joe <example@users.noreply.github.com>"]
-description = ""
+spin_manifest_version = 2
+
+[application]
 name = "hello-world"
-trigger = { type = "http", base = "/" }
-version = "0.1.0"  
-[[component]]
-id = "hello-world"
-source = "target/hello-world.wasm"
-exclude_files = ["**/node_modules"]
-ai_models = ["llama2-chat"]
-[component.trigger]
+version = "0.1.0"
+authors = ["Joe <example@users.noreply.github.com>"]
+description = "My first Serverless AI app"
+
+[[trigger.http]]
 route = "/..."
-[component.build]
+component = "hello-world"
+
+[component.hello-world]
+source = "app.wasm"
+[component.hello-world.build]
 command = "spin py2wasm app -o app.wasm"
-watch = ["app.py", "Pipfile"]
 ```
  
 {{ blockEnd }}
```

Comment:
Can't leave a comment but do you want to update the Rust HTTP handler code?

Reply:
Looks like this is already taken care of. The code here compiles and works with rc1.